bin/blue_green_switch

```diff
@@ -18,29 +16,74 @@ class AWSClient
 end
 
 
-alb = AWSClient.elbv2.describe_load_balancers.load_balancers.find { |x| x.load_balancer_name == lb_name }
-listener = AWSClient.elbv2.describe_listeners(load_balancer_arn: alb.load_balancer_arn).listeners.find { |x| x.port == lb_port }
-rule = AWSClient.elbv2.describe_rules(listener_arn: listener.listener_arn).rules.find { |x| x.conditions.first.values.include? domain }
-tg_green = AWSClient.elbv2.describe_target_groups.target_groups.find { |x| x.target_group_name == "#{tg_prefix}-green" }
-tg_blue = AWSClient.elbv2.describe_target_groups.target_groups.find { |x| x.target_group_name == "#{tg_prefix}-blue" }
-
-if color == "green"
+if color_port == nil
```

Comment:
```
color_port.nil?
```

Reply:
but why do you need this check if `else` part would do the same for `nil` value?

Reply:
if you just need this `if` statement to just print the message you can use this approach:

```
if color
  puts "Switching to #{color}"
else
  fail "...."
end
```

Reply:
or if you want to validate color value:
```
if ["blue", "green"].include?(color)
```

Reply:
Changed. But we still need the color_port check because it will not be handled by `else` part.